### PR TITLE
Repair typos in dates.

### PR DESCRIPTION
**Pull request by arpie-steele**
Fix for #3628 (which explains how corrected dates were determined.)

dfcoeleqvrels, dfcoeleqvrel -> 28-Nov-2022
2mulprm -> 8-Jun-2023

### CODE REVIEW

**Review by wlammen — APPROVED** [review]

**Review by tirix — APPROVED** [review]

**Review by mazsa — CHANGES_REQUESTED** [review]
@arpie-steele : @avekens approved his part of your suggestions https://github.com/metamath/set.mm/issues/3628#issuecomment-1809692002 , please commit that as well.

**Review by mazsa — APPROVED** [review]

**Review by icecream17 — APPROVED** [review]

**Comment by tirix** [comment]
This obviously includes the change in @avekens's theorem date requested by @mazsa, so I will merge without waiting for his approval.

**Pull request merged**
